tests/cases/conformance/importDefer/dynamicImportDefer.ts

```diff
@@ -0,0 +1,14 @@
+// @module: esnext,nodenext,es2020,es2015,commonjs
+// @target: es2020
+
+// @filename: a.ts
+export function foo() {
+  console.log("foo from a");
+}
+
+// @filename: b.ts
+import.defer("./a.js").then(ns => {
+  ns.foo();
+});
+
+import("./a.js"); // TODO: Without this the import.defer cannot resolve ./a
```

Comment:
I'm going to debug this next week, but if you have any pointer as to why I would be getting an error about `./a.js` not being resolved in `import.defer(...)` without this I'd appreciate it :)